### PR TITLE
Tr 2037 Compute msg size after compress

### PR DESCRIPTION
**Pull request by mccammon**
* Compute msg size after compress in extendedSend().
* Fix minor bug where need to compensate for payload size increase after compressing with no compression (~50bytes difference).

### CODE REVIEW

File: test/unit/lib/sqs.spec.js [hunk 1]
```diff
@@ -413,6 +413,84 @@ describe('SQS Utilities', function() {
...
+            MessageBody: compress(payload, {
+              level: expected_compress_level
+            }).toString('base64'),
+            QueueUrl: queueUrl,
+            MessageAttributes: { foo: 'bar' }
+          }); // Expecting to be compressed (>64kb)
+        });
+    });
+
+    it('should send as sqs messages and not compress if compressed payload <=64kb', function() {
```
Comment (mccammon) [inline test/unit/lib/sqs.spec.js]: Oops- wrong name.

Comment (mccammon) [reply test/unit/lib/sqs.spec.js]: fixed.

File: lib/sqs/index.js [hunk 2]
```diff
@@ -11,10 +11,14 @@ const compress = util.promisify(zlib.gzip);
...
 
-// 256kb => b
+// 256kb => b: (256kb * 1024 bytes =  262144)
 const MAX_MSG_SIZE = 262144;
-//64kb => bytes
-const MAX_COMPRESS_MSG_SIZE = 65536;
+/*
+ * Max size of message that does not need to be compressed as it can be sent via 1 SQS Operation
+ *  64kb => b: (64kb * 1024 bytes = 65536 bytes - 50 bytes = 65486)
+ *  -50 bytes to compensate for size increase after compressing with no compression level
```
Comment (ischer) [inline lib/sqs/index.js]: how did we settle on 50 bytes for this buffer?

Comment (mccammon) [reply lib/sqs/index.js]: Series of test up to 64kb never saw more than 40 bytes added post compression with no compression rate.  Added another 10 byte buffer for good measure.

**Pull request merged**